master/static/srv/get_model_versions.sql

```diff
@@ -33,6 +33,7 @@ c AS (
 )
 SELECT
     to_json(c) AS checkpoint,
-    version AS version,
-    creation_time
-    FROM c, m, mv
+    mv.version,
+    mv.creation_time
```

Comment:
Nice, I actually checked a database and missed that `model_versions` has a `creation_time` too. Although, is it the same thing? I don't know the model registry code at all. Is it creation time of the original model versus that of this specific version? If there is a difference, this one seems at least as reasonable to include, but I wanted to check if this represents a change.

Reply:
`model_versions.creation_time` is the creation time of the model version, model's is model's, and the proto for the endpoint says the creation time on the response is supposed to be the former https://github.com/determined-ai/determined/blob/master/proto/src/determined/model/v1/model.proto#L43-L44

Reply:
>  but I wanted to check if this represents a change.

yeah, this does represent a change, but i think this way is correct (esp since it's what the docs say). fwiw, too, the CLI doesn't even surface this field, it seems.

Reply:
Sounds good!